libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -306,6 +306,7 @@ export default function TabBuild() {
       maxBuildsToShow,
     } = buildSetting
     if (!characterKey || !optimizationTarget) return
+    Notification.requestPermission()
```

Comment:
Just a question for my own interest, why do we have to call this request here and it's not enough to do it on the button click handler?

Reply:
So that way if someone has the notification enabled, but permission not granted (they ignored it before, or had it enabled before we did this setup), when they click optimize, it prompts again. Removed the `await` so it doesn't block the optimization